ee/api/explicit_team_member.py

```diff
@@ -120,7 +121,7 @@ def get_permissions(self):
         ):
             # Special case: allow already authenticated users to leave projects
             return []
-        return super().get_permissions()
+        return [permission() for permission in self.permission_classes]
```

Comment:
Won't this override what TeamAndOrgViewSetMixin provides?

Reply:
Yeah there is some special cases like this where they are doing very specific things. Typically only on the "base" models like Team or Organization